truss/_integration-tests/README.md

```diff
@@ -0,0 +1,38 @@
+# integration-tests
+
+These tests run truss against definition files and tests if the generated code
+behaves. 
+
+# ./http
+
+The http directory contains http transport tests. They include tests against
+the generated http transport client libraries against the server and testing
```

Comment:
I find this whole sentence kind of awkward.

> They include tests of
the generated HTTP transport client libraries against the server as well as
testing hand made HTTP requests against the server.

Something like this seems clearer:

> The `http` directory contains tests for the generated HTTP handlers. Both the generated HTTP client and the standard Go `net/http` client are used to test the HTTP handlers.

I don't even know if that's necessary.

> The `http` directory contains tests for the generated HTTP handlers.

Substitute for "handlers" however is more accurate, if necessary.